BUILD.bazel

```diff
@@ -253,6 +253,7 @@ cc_library(
         "@com_google_absl//absl/container:inlined_vector",
         "@com_google_absl//absl/strings",
         "@com_google_absl//absl/strings:str_format",
+        "@com_google_absl//absl/strings:str_join",
```

Comment:
The bazel CI failure is because there isn't a separate `:str_join` target. `str_join.h` is included in the `:strings` target: https://github.com/abseil/abseil-cpp/blob/ac267be5cf9b722a89c0293ccaa7df1ecdb3a446/absl/strings/BUILD.bazel#L84. I think the `"@com_google_absl//absl/strings"` line above should have included it already.

Reply:
Well, I also hoped so, but it failed being unable to find `str_join` in https://github.com/p4lang/p4c/actions/runs/10330407855/job/28599406481

Need to check what's going on more... maybe some includes are missed.

Reply:
Something like https://github.com/p4lang/p4c/blob/0e2a7d622ea94f209542f21810ee16567aac5df8/backends/tc/tc.def#L17-L21 seems to be a solution.

Reply:
Yeah. I'm wondering why this is not required for non-bazel builds though :)

Reply:
Ah, didn't realize this. That's interesting 🤔